spec/views/hearings/_hearing_events.html.haml_spec.rb

```diff
@@ -57,5 +60,49 @@
         .to have_selector('tbody.govuk-table__body tr:nth-child(1)', text: '10:30')
         .and have_selector('tbody.govuk-table__body tr:nth-child(2)', text: '16:30')
     end
+
+    context 'with hearing_events notes' do
+      it 'displays the notes' do
+        is_expected
+          .to have_content('1 hour delay')
+          .and have_content('ended late')
+      end
+
+      context 'with notes containing HTML' do
+        let(:hearing_event_with_html_notes) do
+          hearing_event_class.new(description: 'day 1 start', note: '<b>1 hour delay</b>',
+                                  occurred_at: '2021-01-17T10:30:00.000Z')
+        end
+        let(:hearing_events) { [hearing_event_with_html_notes] }
+
+        it 'displays any HTML raw' do
+          is_expected.to have_content('<b>1 hour delay</b>')
+        end
+      end
+
+      context 'with notes containing unicode characters' do
+        let(:hearing_event_with_unicode_notes) do
+          hearing_event_class.new(description: 'day 1 start', note: '!\"#£%&()*,-./',
```

Comment:
Is this a check for unicode characters?  
Can we include some more characters here?  
`Æ ½ ŵ €`
The ŵ is probably the most important of these 4.

Reply:
And `"" '' –` - the common MS Word Autocorrect characters.

Reply:
Just added the 4 extra characters you suggested, the MS Word characters you mentioned are already included in the test case.